File: documentation/user/en/query/requirements/examples/histogram/attribute-histogram.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				require(
					attributeHistogram(20, STANDARD, "width", "height")
				)
			)
		);
	}
);

File: documentation/user/en/query/requirements/examples/histogram/attribute-histogram.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	Require(
        		AttributeHistogram(20, Standard, "width", "height")
        	)
        )
	)
);